### PR TITLE
add an edit button on Documentation pages

### PR DESCRIPTION
**Pull request by steren**
I successfully tested the change by playing with Chrome DevTools on bazel.build, as I could not find instructions about how to run the site locally :(.

### CODE REVIEW

**Comment by steren** [comment]
screenshot: screen shot 2017-05-11 at 2 56 26 pm

File: site/_layouts/documentation.html [hunk 1]
```diff
@@ -107,6 +107,20 @@
...
         <div class="col-lg-9">
+          <a id="gh-edit" style="visibility: hidden;"><i class="fa fa-pencil" aria-hidden="true"></i> Edit</a>
+          <script>
+            var ghDocsBazeURL = 'https://github.com/bazelbuild/bazel/tree/master/site/versions/master/docs/';
+            var editButton = document.getElementById('gh-edit');
+            // if there is an edit button and we are on a doc page but not in the Build Encyclopedia
+            if(editButton
+              && window.location.pathname.split('docs/').length > 1
+              && window.location.pathname.lastIndexOf('/be/') == -1) {
+              var mdFile = window.location.pathname.split('docs/')[1].replace('html', 'md');
```
Comment (damienmg) [inline site/_layouts/documentation.html]: We still have 4 .html files in that:

$ find site/versions -name '*.html'
site/versions/master/docs/bazel-user-manual.html
site/versions/master/docs/build-ref.html
site/versions/master/docs/query.html
site/versions/master/docs/test-encyclopedia.html


You might want to add special cases for those.

Comment (steren) [inline site/_layouts/documentation.html]: arg yes I did not see.
Couldn't we just rename them to .md?

**Comment by damienmg** [comment]
Not sure, the file are shared with our internal doc. It is probably easier
to special case them.

**Comment by kchodorow** [comment]
You can try the site locally by running scripts/serve-docs.sh.

**Comment by steren** [comment]
@kchodorow yes I can, but then I hit many Jekyll modules missing. How can I install all of them? (from a gem list file?) This has to be documented.

File: site/_layouts/documentation.html (hunk 1 above)
Comment (steren) [inline site/_layouts/documentation.html]: It made me sad to hardcode these pages. But done.

**Comment by kchodorow** [comment]
I have no idea how you're supposed to do this, I think I just did `gem install _whatever_` until it stopped complaining?

**Pull request closed without merging**